docs/building.rst

```diff
@@ -72,21 +72,21 @@ Linux::
 
 	sudo apt install libboost-tools-dev libboost-dev libboost-system-dev
 	echo "using gcc ;" >>~/user-config.jam
-	b2 crypto=openssl cxxstd=14
+	b2 crypto=openssl cxxstd=14 variant=release
```

Comment:
`variant=` is not needed, you can just say `release`. I would prefer that.

Reply:
if you say so.